README.md

```diff
@@ -15,11 +15,20 @@ jobs:
     runs-on: ${{ matrix.os }}
     steps:
       - name: Checkout
-        uses: actions/checkout@v2
+        uses: actions/checkout@v4
       - name: Run headless test
         uses: coactions/setup-xvfb@v1
         with:
           run: npm test
           working-directory: ./ #optional
           options: #optional
 ```
+
+Modern [GitHub Actions Ubuntu images](https://github.com/actions/runner-images/tree/main/images/ubuntu) now include `xvfb` so this Action could be replaced by:
+
+```yml
+- if: runner.os == 'Linux'
+  run: xvfb-run npm test
+- if: ${{ ! runner.os == 'Linux' }}
```

Comment:
For some reason this did not run test on macos and windows, below fixes the problem 

```suggestion
- if: runner.os != 'Linux'
```